lib/mongo/protocol.ex

```diff
@@ -89,6 +90,7 @@ defmodule Mongo.Protocol do
   defp ssl(%{socket: {:gen_tcp, sock}} = s, opts) do
     host      = (opts[:hostname] || "localhost") |> to_charlist
     ssl_opts = Keyword.put_new(opts[:ssl_opts] || [], :server_name_indication, host)
+    ssl_opts = Keyword.put_new(ssl_opts, :ciphers, ['AES256-GCM-SHA384'])
```

Comment:
This should not be the responsibility of the driver. The user can add this to their `:ssl_opts` if the need it.

Reply:
I went back and forth on this, because you're right, but also digging out this arcane tidbit about Erlang's SSL from the depths of the internet was a huge pain. I think a good in-between will be to create an `examples` dir where people can contribute specific connection params that might be non-obvious. I'll add one to my PR. Hopefully this works for you.